packages/liveblocks-node/src/__tests__/client.test.ts

```diff
@@ -221,1555 +221,1730 @@ describe("client", () => {
       }
     });
   });
-
-  test("should return a list of room when getRooms receives a successful response", async () => {
-    const client = new Liveblocks({ secret: "sk_xxx" });
-    await expect(client.getRooms()).resolves.toEqual({
-      nextPage: "/v2/rooms?startingAfter=1",
-      data: [room],
+  describe("get rooms", () => {
```

Comment:
Updated the tests to use a `describe` block as well for easier navigation of the test file as it was getting very large.